transport/src/test/java/io/el/channel/DefaultChannelPipelineTest.java

```diff
@@ -70,4 +81,115 @@ public void removeOnlyOne() {
       assertNull(channelPipeline.firstContext());
     }
   }
+
+  @Nested
+  @DisplayName("On bind() method")
+  class BindMethod {
+    @Test
+    public void checkChannelBindCalled() throws InterruptedException {
+      Executor executor = new ThreadPerTaskExecutor(
+          Executors.defaultThreadFactory());
+      TestEventLoop eventLoop = new TestEventLoop(executor);
+
+      try {
+        final CountDownLatch latch = new CountDownLatch(1);
+        Channel channel = new TestChannel() {
+          @Override
+          public ChannelPromise bind(SocketAddress localAddress) {
+            assert ((InetSocketAddress) localAddress).getPort() == 8080;
```

Comment:
Please use JUnit assertEqual instead of java assert

Reply:
I fixed it in the commit: https://github.com/zeroFruit/el/pull/54/commits/6c971afe8cc439769a30286ff61ecb016a1e6d84